apps/api/test/04-report-4.e2e-spec.ts

```diff
@@ -81,17 +81,17 @@ describe('Report 4 - Number of IEN registrants in the licensing stage', () => {
     await app.close();
```

Comment:
This file was only changed to trigger the PR GitHub actions. Changed to original form here